docs/development.md

```diff
@@ -174,7 +174,7 @@ Here is an example docstring:
 
 ```
 def some_method(a: int, b: str) -> float:
-    r"""One line summary of method.
+    """One line summary of method.
```

Comment:
That `r` was needed! There are backslashes in the doc string. At the moment the docs are going lose the `\\` because it will turn into a `\`, so the matrix will draw wrong.

Reply:
Gotcha! Sorry about that, fixed!